src/main/java/competition/subsystems/shooterwheel/ShooterWheelSubsystem.java

```diff
@@ -72,5 +76,13 @@ public void stop () {
     public void periodic() {
         leader.periodic();
         currentRpmProp.set(leader.getVelocity());
+        dashboardLightProp.set(dashboardLight());
+    }
+
+    public boolean dashboardLight() {
+        if (currentRpmProp.get() >= targetRpmProp.get() - currentToTargetProp.get()) {
```

Comment:
change to `Math.abs(currentRpmProp.get() - targetRpmProp.get()) < currentToTargetProp.get()`